spec/cb/utils/validator_spec.rb

```diff
@@ -14,125 +14,138 @@ module Cb
   describe ResponseValidator do
     let(:response)          { double(HTTParty::Response) }
     let(:response_property) { double('HTTParty::Response.response') }
+    let(:response_body)     { nil }
+    let(:response_code)     { 200 }
 
     before :each do
       allow(response).to receive(:response).and_return(response_property)
-      allow(response).to receive(:code).and_return(200)
+      allow(response).to receive(:code).and_return(response_code)
+      allow(response_property).to receive(:body).and_return(response_body)
```

Comment:
This can all be done in `let(:response)`:
``` ruby
let(:response_body)     { nil }
let(:response_code)     { 200 }
let(:response_property) { double('HTTParty::Response.response', body: response_body) }
let(:response) { instance_double(HTTParty::Response, response: response_property, code: response_code) }

# No more before block needed
```